src/backend/marsha/core/lti/__init__.py

```diff
@@ -107,7 +112,28 @@ def verify(self):
                 )
             )
 
-        return consumer_site
+        self._consumer_site = consumer_site
```

Comment:
`self._is_verified` and `self._consumer_site` are probably redundant because the lti instance is verified when `self._consumer_site is not None`.